### DISCUSSION TITLE
No such file or directory: 'antiderivative_aligned_train.npz' when I want to run the examples/operator/antiderivative_aligned.py

### DISCUSSION THREAD

**Discussion by Liu-xiandong** [lululxvi/deepxde#872] [Q&A]
No such file or directory: 'antiderivative_aligned_train.npz' when I want to run the examples/operator/antiderivative_aligned.py

**Comment by forxltk** [comment]
See https://deepxde.readthedocs.io/en/latest/demos/operator/antiderivative_aligned.html

**Reply by Liu-xiandong** [reply]
Thanks~
